backend/fakedata/generator.js

```diff
@@ -0,0 +1,235 @@
+    	description: casual.description,
+    	author: casual.random_element(context.userId),
+    	dateCreated: casual.date(format = 'YYYY-MM-DD'),
+    	files: casual.random_element(context.fileId),
+    	archived: casual.coin_flip,
+    	isFinished: casual.coin_flip,
+	}
+})
+
+casual.define('toDo', function(){
+	var id = casual.m_id;
+	context.todoId.push(id);
+	var task = casual.task;
+	untrackted.task.push(task);
+	return{
+		_id: id,
+	    task: task._id,
+   		title: casual.title,
+   		description: casual.description,
+   		status: casual.word,
+   		participants: [casual.random_element(context.userId)],
+   		dateExpired: casual.date(format = 'YYYY-MM-DD'),
+   		dateFinished: casual.date(format = 'YYYY-MM-DD')
+	}
+});
+
+
+casual.define('event', function(){
+	var id = casual.m_id;
+	context.eventId.push(id);
+	var part = [];
+	part.push(casual.random_element(context.userId));
+	part.push(casual.random_element(context.userId));  
+	var files = [];
+	files.push[casual.random_element(context.fileId)];
+	return {
+		_id: id,
+	    title: casual.title,
+    	description: casual.description,
+    	project: casual.random_element(context.projectId),
+    	isAllDay: casual.coin_flip ,
+    	startDate: casual.date(format = 'YYYY-MM-DD'),
+    	endDate: casual.date(format = 'YYYY-MM-DD'),
+    	participants: part,
+    	// files: [{
+    	//     type: Schema.Types.ObjectId,
+    	//     ref: "File"
+    	// }]
+	}
+});
+
+casual.define('message', function(){
+	var id = casual.m_id;
+	context.messageId.push(id);
+	var comm = [];
+	comm.push({author:casual.random_element(context.userId), date: casual.date(format = 'YYYY-MM-DD'), 
+		description: casual.description, files: casual.word})
+	return {
+		_id: id,
+	    isDraft: casual.coin_flip ,
+    	title: casual.title,
+    	description: casual.description,
+    	author: casual.random_element(context.userId),
+    	project: casual.random_element(context.projectId),
+    	date: casual.date(format = 'YYYY-MM-DD'),
+    	comments: comm,
+    	// [
+    	//     {
+    	//         author: {
+    	//             type: Schema.Types.ObjectId,
+    	//             ref: 'User'
+    	//         },
+    	//         date: Date,
+    	//         description: String,
+    	//         files: [String]
+    	//     }
+    	// ],
+    	files: casual.word
+	}
+});
+
+casual.define('project', function(){
+	var id = casual.m_id;
+	context.projectId.push(id);
+	var part = [];
+	part.push(casual.random_element(context.userId));
+	part.push(casual.random_element(context.userId));            ///   can be the same!!!!!!!!!!!!!!!
+	return {
+		_id: id,
+        title: casual.title,
+        description: casual.desctription,
+        participants: part,
+        startDate: casual.date(format = 'YYYY-MM-DD'),
+        endDate: casual.date(format = 'YYYY-MM-DD'),
+        version: casual.century,                                  
+        repository: String
+    }
+})
+
+
+function fake(){
+
+	for (var i=0; i<3; i++){

```

Comment:
it is better to implement this cycle in separate function that takes service, casual entity and count to be generated. And fake function just call these functions few times with different entities.